src/index.html

```diff
@@ -11,7 +11,98 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form__field">Surname:
+          <input type="text" 
+          autocomplete="off" 
+          name="surname"
+          >
+        </label>
+        <label class="form__field">Name:
+          <input type="text"
+           autocomplete="off" 
+           name="name"
+          >
+        </label>
+        <label class="form__field">How old are You?
+          <input type="number"
+          value="12" name="age"
+          min="1" max="100"
+          >
+        </label>
+        <label class="form__field">Full date of birth:
+          <input type="date" name="birth_date">
+        </label>
+        <label class="form__field">I accept the term of the agreement
+          <input type="checkbox" name="agreed">
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
```

Comment:
```suggestion
      </fieldset>
      
      <fieldset class="fieldset">
```
add spaces between blocks of code for better readability